### PR TITLE
[kibana] ensure the latest version 7.x works including new image

### PR DESCRIPTION
**Pull request by alejandroEsc**
**What type of PR is this?**

chore

**What this PR does/ why we need it**:

Changes needed to ensure new version of kibana works and behaves properly

**Which issue(s) this PR fixes**:

https://jira.d2iq.com/browse/D2IQ-73499

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
Kibana chart updated to ensure we can deploy version 7.10.2
```

**Checklist**

* [ ] *If a chart is changed, the chart version is correctly incremented.*
* [ ] The commit message explains the changes and why are needed.
* [ ] The code builds and passes lint/style checks locally.
* [ ] The relevant subset of integration tests pass locally.
* [ ] The core changes are covered by tests.
* [ ] The documentation is updated where needed.

### CODE REVIEW

File: stable/kibana/templates/configmap-dashboardimport.yaml [hunk 1]
```diff
@@ -23,21 +23,22 @@ data:
 
     echo "Waiting up to {{ .Values.dashboardImport.timeout }} seconds for Kibana to get in green overall state..."
 
+    # This could change in the future so we should monitor this on each release
     for i in {1..{{ .Values.dashboardImport.timeout }}}; do
-      curl -s localhost:5601{{ .Values.dashboardImport.basePath }}/api/status | python -c 'import sys, json; print json.load(sys.stdin)["status"]["overall"]["state"]' 2> /dev/null | grep green > /dev/null && break || sleep 1
+      curl -s localhost:5601{{ .Values.dashboardImport.basePath }}/api/status | grep "Looking good" 2> /dev/null | grep green > /dev/null && break || sleep 1
```
Comment (joejulian) [inline stable/kibana/templates/configmap-dashboardimport.yaml:28]: Would this work?
```suggestion
      curl -s -m1 localhost:5601{{ .Values.dashboardImport.basePath }}/api/status | grep -q "Looking good.*green" && break || sleep 1
```

**Review by joejulian — APPROVED** [review]

File: stable/kibana/values.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 image:
-  repository: "docker.elastic.co/kibana/kibana-oss"
-  tag: "6.7.0"
+  repository: "docker.elastic.co/elasticsearch/elasticsearch"
+  tag: "7.10.2-SNAPSHOT"
```
Comment (samvantran) [inline stable/kibana/values.yaml:3]: Should we use `7.10.1` instead of a SNAPSHOT? That's usually a dev tag

**Comment by alejandroEsc** [comment]
Tests are failing since they seem to require that we deploy elasticsearch.

**Comment by alejandroEsc** [comment]
\test-helm3

**Review by s12chung — APPROVED** [review]

File: stable/kibana/values.yaml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 image:
   repository: "docker.elastic.co/kibana/kibana-oss"
-  tag: "6.7.0"
+  tag: "7.9.3"
```
Comment (gracedo) [inline stable/kibana/values.yaml:3]: should this be `7.10.1` (what `appVersion` is set to)?
https://github.com/mesosphere/charts/pull/933/files#diff-38a0491093d5cf5568e8c1f59ecd4669b929d03e8c476864bcddf31e6d4abf46R4

Comment (alejandroEsc) [reply stable/kibana/values.yaml:3]: No, we cannot upgrade to 7.10.1 unfortunately, the plugin for it is currently broken, we can change this when those problems have been resolved.

Comment (gracedo) [reply stable/kibana/values.yaml:3]: ok got it thanks, let's update the appVersion then

File: stable/kibana/Chart.yaml [hunk 4]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 name: kibana
-version: 3.2.8
-appVersion: 6.7.0
-description: DEPRECATED - Kibana is an open source data visualization plugin for Elasticsearch
+version: 3.3.0
+appVersion: 7.10.1
```
Comment (gracedo) [inline stable/kibana/Chart.yaml:4]:
```suggestion
appVersion: 7.9.3
```

**Review by gracedo — APPROVED** [review]

**Pull request merged**